CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Misc:
 
 Breaking changes:
 - [#1662](https://github.com/rails-api/active_model_serializers/pull/1662) Drop support for Rails 4.0 and Ruby 2.0.0. (@remear)
+- [#1720](https://github.com/rails-api/active_model_serializers/pull/1720) Block relationships must explicitly use `load_data` instead of the return value to customize relationship contents.
```

Comment:
@richmolj I don't want to ask you to do a lot more work at this point, but would you mind sketching out some docs with usage, perhaps modify existing docs? (I'm just going through issues and am not reading all the code, and I didn't see a clear example.. forgive me if it's clear somewhere and I missed it)